extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -694,4 +719,100 @@ export class AmpAnalytics extends AMP.BaseElement {
   }
 }
 
+/**
+ * Keeps track of the third-party vendors' cross-domain iframes. Only one
+ * static instance will be created, which will keep manage ALL amp-analytics
+ * tags.
+ */
+class AmpAnalyticsFrames {
+  constructor() {
+    this.map = new Map();
+  }
+
+  /**
+   * Create a cross-domain iframe
+   * @param {!Element} parent  The document node of the parent page
+   * @param {!string} requestsFrameUrl  The URL, including data hash if
+   * applicable, of the cross-domain iframe
+   * @return {!Element}
+   */
+  create(parent, requestsFrameUrl) {
+    const context = {
+      "scriptSrc": "/examples/analytics-3p-remote-frame-helper.js",
+    };
+    let frame = createElementWithAttributes(parent, 'iframe', {
+      sandbox: 'allow-scripts',
+      name: JSON.stringify(context)
```

Comment:
Good catch. In a previous iteration, context contained a lot more stuff. Now that it's just one key/val, absolutely doesn't need to be extracted into a var like that. Thanks!